sources/packages/backend/libs/services/src/workflow/workflow-client.service.ts

```diff
@@ -171,4 +171,28 @@ export class WorkflowClientService {
       // The error is not thrown here, as we are failing silently.
     }
   }
+
+  /**
+   * After the assessment calculations are completed and persisted
+   * send message to unblock the next assessment which is waiting for this assessment
+   * in the sequence to be calculated.
+   * @param assessmentId assessment waiting for calculation.
+   */
+  async sendAssessmentCalculationCompleteMessage(
+    assessmentId: number,
+  ): Promise<void> {
+    try {
+      await this.zeebeClient.publishMessage({
+        name: "assessment-calculation-completed",
```

Comment:
I am having a hard time figuring out if the name would be easier to understand. It may cause some confusion because the calculation being completed is actually from another assessment workflow.
Should we have it renamed to something that would indicate better that it is from outside the workflow itself? For instance: `related-assessment-calculation-completed`.